node/middlewares/prepare.ts

```diff
@@ -1,15 +1,15 @@
 import { parse } from 'query-string'
 
 import { BindingResolver } from '../resources/bindings'
-import { CONFIG_BUCKET, CONFIG_FILE, getBucket, getMatchingBindings, hashString } from '../utils'
-import { GENERATE_SITEMAP_EVENT } from './generateMiddlewares/utils'
+import { CONFIG_BUCKET, CONFIG_FILE, getBucket, getMatchingBindings, hashString, startSitemapGeneration } from '../utils'
+import { DEFAULT_CONFIG } from './generateMiddlewares/utils'
 
 // const ONE_DAY_S = 24 * 60 * 60
```

Comment:
can u remove this?
```suggestion
```

Reply:
I prefer to keep it so that we won't forget wot change the cache control back to one day. I will add a TODO to make it explicit.